activerecord/test/cases/log_subscriber_test.rb

```diff
@@ -254,6 +256,12 @@ def test_where_in_binds_logging_include_attribute_names
       assert_match(%{["id", 1], ["id", 2], ["id", 3], ["id", 4], ["id", 5]}, @logger.logged(:debug).last)
     end
 
+    def test_where_in_binds_with_json_attribute
+      Admin::User.where(json_options: [{ a: 1 }, { b: 2 }]).load
+      wait
+      assert_match(%{[["json_options", "{\\"a\\":1}"], ["json_options", "{\\"b\\":2}"]]}, @logger.logged(:debug).last)
+    end
```

Comment:
This tests the logging output, but what we're really trying to fix is the query, so I think we should test the query results instead.